checkov/terraform/checks/resource/gcp/GoogleComputeShieldedVM.py

```diff
@@ -0,0 +1,32 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class GoogleComputeShieldedVM(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure Compute instances are launched with Shielded VM enabled"
+        id = "CKV_GCP_39"
+        supported_resources = ['google_compute_instance']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        """
+        Shielded VM can only be used with boot images with shielded VM support.
+        See the complete list of supporting images here: https://cloud.google.com/compute/docs/images#shielded-images
+        :param conf:
+        :return: <checkResult>
+        """
+
+        if 'shielded_instance_config' in conf.keys():
+            if 'enable_vtpm' in conf['shielded_instance_config'][0] and \
+                    not conf['shielded_instance_config'][0].get('enable_vtpm')[0]:
```

Comment:
@arielkru `something.get('enable_vtpm')[0]` will fail if something doesn't have any value for `enable_vtpm`. please have the get return `[]` by default

Reply:
It must have a value for it because the `if` first checks if the key `enable_vtpm` exists. actually there's no need to use the get function because I first check if the key exists, so I removed it.